### PR TITLE
Patch to fix crash when putting on HMD for Qt 5.9

### PR DESCRIPTION
**Pull request by SamGondelman**
Addresses fogbugz [5540](https://highfidelity.fogbugz.com/f/cases/5540/Qt-5-9-Only-donning-oculus-when-interface-is-already-in-VR-mode-crashes-interface)

Fixes/bypasses a crash in OculusDisplayPlugin::hmdPresent() when putting on a Rift while already in VR mode as a result of an "invalid ovrSession."

Test plan:
- Gustavo's Qt 5.6 build should behave the same as master.  Should not crash if you start Interface in VR mode with the Rift off and then put the Rift on.
- If you build from scratch with Qt 5.9, should no longer crash when you do that.

### CODE REVIEW

**Comment by SamGondelman** [comment]
Note:
This seems indicative of a larger underlying problem associated with Qt 5.9 and something about our framebuffers, but I'm putting this up in the interest of patching the crash.

Without this change, before crashing, the log would display:
`14/07 14:37:49.087 {!ERROR!} [Kernel:Error] OVR Error:
  Code: -1002 -- ovrError_InvalidSession
  Description: Invalid ovrSession provided.
  OVRTime: 348019.829390
  Time: 2017-07-14 14:37:49 [87:668:800]

14/07 14:37:49.087 {!ERROR!} [Kernel:Error] OVR Error:
  Code: -1002 -- ovrError_InvalidSession
  Description: Invalid ovrSession provided.
  OVRTime: 348019.829507
  Time: 2017-07-14 14:37:49 [87:784:900]

[07/14 14:37:49] [DEBUG] [hifi.gl] OpenGL:  GL_INVALID_FRAMEBUFFER_OPERATION error generated. Operation is not valid because a bound framebuffer is not framebuffer complete.
14/07 14:37:49.087 {!ERROR!} [Kernel:Error] OVR Error:
  Code: -1002 -- ovrError_InvalidSession
  Description: Invalid ovrSession provided.
  OVRTime: 348019.829829
  Time: 2017-07-14 14:37:49 [88:107:700]

14/07 14:37:49.087 {!ERROR!} [Kernel:Error] OVR Error:
  Code: -1002 -- ovrError_InvalidSession
  Description: Invalid ovrSession provided.
  OVRTime: 348019.829933
  Time: 2017-07-14 14:37:49 [88:211:0]

[07/14 14:37:49] [WARNING] [hifi.plugins.display.oculus] Failed to present : Invalid ovrSession provided.
14/07 14:37:49.087 {!ERROR!} [Kernel:Error] OVR Error:
  Code: -1002 -- ovrError_InvalidSession
  Description: Invalid ovrSession provided.
  OVRTime: 348019.830118
  Time: 2017-07-14 14:37:49 [88:395:800]`

Now, the log displays:
`14/07 15:15:30.286 {!ERROR!} [Kernel:Error] OVR Error:
  Code: -1002 -- ovrError_InvalidSession
  Description: Invalid ovrSession provided.
  OVRTime: 350280.999574
  Time: 2017-07-14 15:15:30 [286:636:700]`

Notably, one is less than five, but both one and five are greater than zero.

**Comment by hifi-gustavo** [comment]
Mac OS X build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/1703/HighFidelity-Beta-PR10971-40658e875e995aefb29f7decfc09073c1d706951.dmg

**Comment by hifi-gustavo** [comment]
Windows build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/1703/HighFidelity-Beta-PR10971-40658e875e995aefb29f7decfc09073c1d706951.exe

File: plugins/oculus/src/OculusDisplayPlugin.cpp [hunk 1]
```diff
@@ -198,7 +198,7 @@ void OculusDisplayPlugin::hmdPresent() {
         static int compositorDroppedFrames = 0;
         static int appDroppedFrames = 0;
         ovrPerfStats perfStats;
-        ovr_GetPerfStats(_session, &perfStats);
+        if (!OVR_SUCCESS(ovr_GetPerfStats(_session, &perfStats))) return;
```
Comment (ZappoMan) [inline plugins/oculus/src/OculusDisplayPlugin.cpp]: if we were going to merge this, we'd need to format it correctly according to coding standards

**Comment by hifi-gustavo** [comment]
Mac OS X build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/1706/HighFidelity-Beta-PR10971-005b1bf9a31866023eeb8a0bc6132ad3b07d7d5b.dmg

**Comment by hifi-gustavo** [comment]
Windows build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/1706/HighFidelity-Beta-PR10971-005b1bf9a31866023eeb8a0bc6132ad3b07d7d5b.exe

**Pull request closed without merging**